packages/caliper-fabric/test/connector-versions/v1/WalletFacade.js

```diff
@@ -0,0 +1,141 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+chai.should();
+const mockery = require('mockery');
+
+/**
+ * simulate a node sdk v1 wallet
+ */
+class StubWallet {
+    /**
+     * Mock V1 wallet implementation
+     */
+    constructor() {
+        this.map = new Map();
+    }
+
+    /**
+     *
+     * @param {*} key b
+     * @param {*} value b
+     */
+    async import(key, value) {
+        this.map.set(key, value);
+    }
+
+    /**
+     *
+     * @param {*} key b
+     */
+    async export(key) {
+        return this.map.get(key);
+    }
+
+    /**
+     *
+     */
+    async list() {
```

Comment:
change this from `list` to `getAllLabels` as that api is the one we want to be using in the real wallet